app/controllers/container_node_controller.rb

```diff
@@ -14,8 +14,8 @@ def show_list
 
   def textual_group_list
     [
-      %i[properties container_labels compliance miq_custom_attributes],
```

Comment:
I thought that `miq_custom_attributes` were our "internal" tags, `has_many   :miq_custom_attributes, -> { where(:source => 'EVM') }` not ones set by the provider.

Did you drop this because you were seeing duplicates?

Reply:
Yeah the custom attribute values were just labels + annotations. Should I readd it?

Reply:
I would have expected miq_custom_attributes to only show custom_attributes with source=EVM, I wonder if there's a bug in there?

It looks like from your original picture the Custom Attributes and Labels were already duplicated, ideally I think we'd want to keep container_labels, miq_custom_attributes, and annotations and just fix miq_custom_attributes.

Reply:
The problem is that `textual_miq_custom_attributes` is wrong [[ref](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/container_summary_helper.rb#L80-L81)]. It is actually calling `custom_attributes` (the superset of all attributes), instead of `miq_custom_attributes` (the subset we expect).  So yes, keep miq_custom_attributes here, but then fix `textual_miq_custom_attributes`